Modules/CodeTrek/Services/CodeTrekService.php

```diff
@@ -17,8 +17,8 @@ public function getCodeTrekApplicants($data = [])
         if ($centre) {
             $applicants = $query->where('centre_id', $centre);
         }
-        $applicants = $search ? $query->whereRaw("CONCAT(first_name, ' ', last_name) like '%$search%'")
-            ->get() : $query->get();
+        $applicants = $search ? $query->whereRaw("CONCAT(first_name, ' ', last_name) like '%$search%'")->get() : $query->get();
+        $applicants = $query->paginate(config('constants.pagination_size'));
```

Comment:
Can we write like this?
Please test locally, and check all the edge cases. And let me know, it's works or not.
```suggestion
       $applicants = $query->when($search, function ($query) use ($search) {
    return $query->whereRaw("CONCAT(first_name, ' ', last_name) LIKE '%$search%'");
})
->paginate(config('constants.pagination_size'));
```

Reply:
@Ayush8923 Yes it is working fine.